src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerVirtualNetworkRule.md

```diff
@@ -0,0 +1,129 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlServerVirtualNetworkRule
+
+## SYNOPSIS
+Gets or lists Azure SQL Server Virtual Network Rule.
+
+## SYNTAX
+
+```
+Get-AzureRmSqlServerVirtualNetworkRule [-VirtualNetworkRuleName <String>] -ServerName <String>
+ [-ResourceGroupName] <String> [-WhatIf] [-Confirm]
+```
+
+## DESCRIPTION
+This command gets a specific Azure SQL Server Virtual Network Rule or a list of Azure SQL Server Virtual Network Rules under a server.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> $virtualNetworkRule = Get-AzureRmSqlServerVirtualNetworkRule -ResourceGroupName rg -ServerName serverName -VirtualNetworkRuleName virtualNetworkRuleName
+```
+
+Gets an single Azure SQL Server virtual network rule
+
+### Example 2
+```
+PS C:\> $virtualNetworkRules = Get-AzureRmSqlServerVirtualNetworkRule -ResourceGroupName rg -ServerName serverName
+```
+
+Gets the list of Azure SQL Server virtual network rules under the specified server
+
+
+## PARAMETERS
+
+### -ResourceGroupName
+The name of the resource group.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ServerName
+The Azure Sql Server name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -VirtualNetworkRuleName
+The Azure Sql Server Virtual Network Rule name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+## INPUTS
+
+### System.String
+
+
+## OUTPUTS
+
+### System.Object
```

Comment:
@kisantia the markdown help will need to be updated whenever you add the output attribute to each cmdlet